Sources/MorphingShapes/MorphingBackground.swift

```diff
@@ -9,6 +9,7 @@ import SwiftUI
 import Foundation
 import Combine
 
+@available(macOS 11.0, *)
```

Comment:
- [ ] Currently, it's version 10.15 in `Package.swift`. Are you sure you need to set macOS version this high?